src/backend/env/post.rs

```diff
@@ -710,13 +750,20 @@ pub fn archive_cold_posts(state: &mut State, max_posts_in_heap: usize) -> Result
                 .posts
                 .remove(&post_id)
                 .ok_or(format!("no post found for id={post_id}"))?;
-            state.memory.posts.insert(post_id, post)
+            #[cfg(test)]
```

Comment:
I think we can avoid it by initializing `POST` in test. Stable structures in test mode should use an in-memory storage automatically.

Reply:
Or is it done to avoid modifying the tests? I am confused.

Reply:
I realized later that this whole test gets removed later in the second PR, as it tests more the allocator logic than anything else. So If you don't mind, I don't see a need to change here anything. Or would you feel more confident without compilation flags?

Reply:
If that specific test is not useful anymore, should we marked it as ignored? To make sure that we don't accidentally disable other important tests.

Reply:
Good idea, will add!